src/scripts/main.js

```diff
@@ -1,7 +1,47 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const elementParent = document.querySelector('body');
+  const notificationElement = document.createElement('div');
+
+  const titleEl = document.createElement('h2');
+
+  titleEl.classList.add('title');
+  titleEl.textContent = title;
+  notificationElement.appendChild(titleEl);
+
+  const paragraph = document.createElement('p');
+
+  paragraph.classList.add('description');
+  paragraph.textContent = description;
+  notificationElement.appendChild(paragraph);
+
+  elementParent.appendChild(notificationElement);
+
+  notificationElement.classList.add('notification');
+  notificationElement.style.boxSizing = 'content-box';
+
+  if (type === 'success') {
+    notificationElement.style.top = `${posTop}px`;
+    notificationElement.style.right = `${posRight}px`;
```

Comment:
so position is the same? it can be removed from `if` to not duplicate.
Also use `switch .. case` instead of 3 `if` and just change class